calen.rb

```diff
@@ -0,0 +1,243 @@
+require '/home/suhaib/Desktop/ems/even'
+$events = {}
+########################## Claender Class ########################## 
+class Calender
+    
+    attr_accessor :date
+
+    def initialize(date)
+        @date = date
+    end
+##########################  Add Event ########################## 
+     
+def self.add_event
```

Comment:
We should take all the inputs in driver class and then call add_event with the args.
OOP violation.